src/utils/indexer/index.ts

```diff
@@ -198,10 +199,15 @@ export async function saveDepositLogs(
   transferMap: Map<string, string>,
 ): Promise<void> {
   let transfer = await transferRepository.findByNonceToDomainId(decodedLog.depositNonce, decodedLog.toDomainId)
+
+  const { sender } = decodedLog
+
+  const addressStatus = await checkSanctionedAddress(sender)
```

Comment:
How many requests do we actually have and will this overload that?
If it might better to do it as a period check of accounts.